GOVERNANCE.md

```diff
@@ -0,0 +1,132 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, lead, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+At the top of the hierarchy are the **Maintainers**. We consider maintainers to
+be the people who contribute, review, guide, and collectively make decisions
+about the direction and scope of the project.
+
+A typical maintainer is not only someone who has made "large" contributions, but
+someone who has shown they are continuously committed to the project and its
+community. Some expected responsibilities of maintainers include (but are not
+exclusively limited to):
+
+- Displaying a high level of commitment to the project and its community, and
+being a role model for others.
+- Writing code &mdash; a lot of code, especially "glue code" or "grunt work";
+fixing bugs, improving processes, making judgments on dependencies, handling
+security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+or long-term vision.
+- When necessary, participate in a collective voting process to resolve
+conflicts and make decisions for which there is no clear answer.
+
+This is not an exhaustive list, nor is it intended that every maintainer does
+each and every one of these individual tasks to equal amounts. Rather this is
+only a guideline for what maintainers are expected to conceptually do.
+
+In short, Maintainers are the outwardly visible stewards of the project.
+
+#### Current list of maintainers
+
+The current list of maintainers:
+
+- Yuya Nishihara
+- Martin von Zweigbergk
+- TBD
+
+TODO: Pending on the initial process of open nominations, decided by Yuya and
+Martin, more names will be added.
+
+### Contributors

```

Comment:
The Contributor role has no authority associated with it, and I don't think we plan to have a record anywhere of who is considered a Contributor and who is not. So I think this section is mostly about saying that we appreciate people more if they are more active in the project. Should we clarify that here (i.e. that it's not a formal role)? Do people find it useful to have this definition of the informal role?

(Unlike the other nits on this doc, please leave this one open so people can comment, @thoughtpolice)

Reply:
> The Contributor role has no authority associated with it, and I don't think we plan to have a record anywhere of who is considered a Contributor and who is not.

It is useful for a [Rust thanks][thx] like website and to distinguish some longtime contributors from passerby's.  

> So I think this section is mostly about saying that we appreciate people more if they are more active in the project. Should we clarify that here (i.e. that it's not a formal role)?

I think this will work for a few years until the project has a few large components (server, web-frontend, etc), where _knowing_ who to contact for a design or for knowledge behind previous decisions will be useful. 

[thx]: https://thanks.rust-lang.org/

> Do people find it useful to have this definition of the informal role?

IMO, it is useful to have _if_ the project wants to do something with it long-term.

Reply:
Also this doesn't between social moderation roles yet (the current maintainer role in Discord and on GitHub [#2227]), which is something I think should also be defined here.